test/jdk/javax/swing/regtesthelpers/JRobot.java

```diff
@@ -153,11 +171,14 @@ public void clickMouse(int buttons) {
      * Perform a click with the first mouse button.
      */
     public void clickMouse() {
-        clickMouse(InputEvent.BUTTON1_MASK);
+        clickMouse(InputEvent.BUTTON1_DOWN_MASK);
     }
 
     /**
      * Click in the center of the given Component
```

Comment:
```suggestion
     * Click in the center of the given Component.
```
Let's fix the javadoc even further and add the full stop.